templates/definition/charger/ocpp.yaml

```diff
@@ -1,25 +1,46 @@
 template: ocpp
 products:
   - description:
-      de: OCPP v1.6 kompatible Wallbox mit Smart Charging Profile
-      en: OCPP v1.6 compatible charger with smart charging profile
+      de: OCPP v1.6 Websocket/JSON kompatible Wallbox mit Smart Charging Profile
```

Comment:
Welcher Anwender soll das verstehen?

Reply:
Ich hab mir die diversen OCPP-Varianten nicht ausgedacht :)
Habe nur versucht deutlich zu machen dass nur "OCPP" oder "OCPP 1.6" nicht ausreicht weil es da wieder diverse Varianten gibt.
Besserer Vorschlag?

Reply:
Ich würds lassen wie es ist. Mir ist in der Praxis noch nix Anderes als ws/json untergekommen. Ich habs eher von Userseite gesehen- Websocket/JSON wird auf keiner Box drauf stehen....

Reply:
OK, ich hatte mich durch die OOCP-Konfigurationsmenü-Screenshots von diversesten Boxen gebrowst und den Eindruck gewonnen dass dies vielleicht etwas helfen könnte.
Aber du hast wohl Recht und es wird ja auch noch bei den Voraussetzungen genannt.

Reply:
Was hältst du von
```suggestion
      de: OCPP 1.6J kompatible Wallbox mit Smart Charging Profile
```